client/web/src/components/FilteredConnection/OrderControl.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useCallback } from 'react'
+
+import { Select, Text } from '@sourcegraph/wildcard'
+
+import { RadioButtons } from '../RadioButtons'
+
+import styles from './OrderControl.module.scss'
+
+export interface OrderedConnectionOrderValue {
+    value: string
+    label: string
+    tooltip?: string
+    args: { [name: string]: string | number | boolean }
+}
+
+/**
+ * An order selector to display next to the search input field.
+ */
+export interface OrderedConnectionOrderingOption {
+    /** The UI label for the ordering option. */
+    label: string
+
+    /** "radio" or "select" */
+    type: string
```

Comment:
😱 

```suggestion
    type: "radio" | "select"
```

but I think we can drop "radio"?